src/io/tap.c

```diff
@@ -48,15 +48,13 @@ static void print_pre_suite(FILE *f, struct criterion_suite_stats *stats) {
             stats->suite->name);
 }
 
-static INLINE bool is_disabled(struct criterion_test *t, struct criterion_suite *s) {
-    return t->data->disabled || (s->data && s->data->disabled);
-}
 
 static void print_test_normal(FILE *f, struct criterion_test_stats *stats) {
     const char *format = can_measure_time() ? "%s - %s::%s %s (%3.2fs)\n"
                                             : "%s - %s::%s %s\n";
     fprintf(f, format,
-            stats->failed ? "not ok" : "ok",
+            stats->test_status == FAILED ? "not ok" :
+                stats->test_status == PASSED ? "ok" : "SKIP",
```

Comment:
this does not follow the [TAP spec](https://testanything.org/tap-version-13-specification.html). For that matter, no need to change anything here -- the SKIPPED case is already handled [L98](https://github.com/Snaipe/Criterion/pull/136/files#diff-0e1bc6cba7767f4d0c3a8eea0dcd5a9aR98)